mantis-common/src/main/java/io/mantisrx/server/core/ServiceRegistry.java

```diff
@@ -34,6 +34,7 @@ private ServiceRegistry() {
     }
 
     public void setMantisPropertiesService(MantisPropertiesLoader service) {
+        logger.info(String.format("Setting Mantis Properties Service to %s", service), new Exception());
```

Comment:
nit: why log an empty exception? also,
`logger.info("Setting Mantis Properties Service to {}", service, new Exception())`